src/Services/Testing/Testing.API/DTOs/DictionaryData/DifficultyData.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Testing.API.DTOs.DictionaryData;
+
+public class DifficultyData
```

Comment:
DifficultyData and etc must not be in DTO folder because it is work model not transfer